en/docs/integrate/examples/data_integration/data-input-validator.md

```diff
@@ -95,9 +95,7 @@ Let's send a request with invalid and valid data to the data service:
     </dat:addEmployeeOp>
     ```
     
-4. Invoke the **addEmployeeOp** operation.
-
-   A validation error is thrown as the response because the **addEmployeeOp** operation has failed. This is because                  the FirstName only has 2 characters.
+4. Invoke the **addEmployeeOp** operation. A validation error is thrown as the response because the **addEmployeeOp** operation has failed. This is because                  the FirstName only has 2 characters.
```

Comment:
```suggestion
4. Invoke the **addEmployeeOp** operation. A validation error is thrown as the response because the **addEmployeeOp** operation has failed. This is because the FirstName only has 2 characters.
```